--- ./Config-0.3.1/Config.php.org	Tue Jun 25 16:40:04 2002
+++ ./Config-0.3.1/Config.php	Fri Jul 12 17:12:45 2002
@@ -249,7 +249,7 @@
                 else 
                     $error = $this->container->parseInput($datasrc,$feature);        
                 
-                if (Pear::isError($error))
+                if (PEAR::isError($error))
                    return $error;
          
                 $totaldata = $this->array_merge_clobber($totaldata,$this->container->data);
@@ -265,7 +265,7 @@
             else 
                 $error = $this->container->parseInput($files,$feature);                    
             
-            if (Pear::isError($error))
+            if (PEAR::isError($error))
                  return $error;
 
             $this->data = $this->container->data;
--- ./Config-0.3.1/README.Config.org	Sat May 25 14:54:11 2002
+++ ./Config-0.3.1/README.Config	Fri Jul 12 17:13:15 2002
@@ -322,7 +322,7 @@
 DB
 
 you can store you're config data also in a Database. This implementation uses
-the Pear::DB Classes, so any Database supported within pear should be supported
+the PEAR::DB Classes, so any Database supported within pear should be supported
 here also.
 
 the table in the Database has to look the following way (the length of the
@@ -485,4 +485,4 @@
 
 There are no special features to be set in parseInput - this argument is
 there for compability reasons. Preserving behaviour during file writing
-is also ignored.
\ No newline at end of file
+is also ignored.
